packages/p2p/src/components/my-ads/buy-ad-payment-methods-list.jsx

```diff
@@ -0,0 +1,202 @@
+                                            autoComplete='off' // prevent chrome autocomplete
+                                            data-lpignore='true'
+                                            list_items={payment_methods_list}
+                                            onItemSelection={({ value }) => onEditPaymentMethodItem(value, key)}
+                                            onFocus={() => {
+                                                setSelectedEditMethod({ value: payment_method, text: method });
+                                                setFieldValue('payment_method', '');
+                                            }}
+                                            onBlur={e => {
+                                                e.preventDefault();
+                                                const value = checkValidPaymentMethod(e.target.value);
+                                                if (e.target.value === '') {
+                                                    setFieldValue('payment_method', method);
+                                                } else if (!value) {
+                                                    onClickDeletePaymentMethodItem(payment_method);
+                                                } else {
+                                                    onEditPaymentMethodItem(value, key);
+                                                }
+                                            }}
+                                            leading_icon={
+                                                <Icon
+                                                    icon={
+                                                        payment_method_icon === 'BankTransfer' ||
+                                                        payment_method_icon === 'Other'
+                                                            ? `IcCashier${payment_method_icon}`
+                                                            : 'IcCashierEwallet'
+                                                    }
+                                                />
+                                            }
+                                            trailing_icon={
+                                                <Icon
+                                                    icon='IcDelete'
+                                                    onClick={() => onClickDeletePaymentMethodItem(payment_method)}
+                                                />
+                                            }
+                                            type='text'
+                                        />
+                                    )}
+                                </Field>
+                            )}
+                        </Formik>
+                    );
+                })}
+                {my_ads_store.payment_method_names.length < 3 && payment_methods_list.length > 0 && (
```

Comment:
```suggestion
                {my_ads_store.payment_method_names.length < 3 && payment_methods_list.length > 0 && (
```
Heyo~ I'm seeing a lot where this `magic number` exists. Can you put it in a constant with a descriptive name so that we know what does it do and why is it set to `3`

Some references for magic numbers and why to avoid them: https://www.pluralsight.com/tech-blog/avoiding-magic-numbers/